### DISCUSSION TITLE
Check Multiplatform Radio App at my account using this library

### DISCUSSION THREAD

**Discussion by jaksatomovic** [fethica/FRadioPlayer#68] [Show and tell]
I have made a multiplatform app using swift UI and this library so feel free to check it out and maybe use it as an example

https://github.com/jaksatomovic/Signal

**Comment by fethica** [comment]
The app looks amazing! Very good job @jaksatomovic 

I will link to it in the README so people can check it out!

**Reply by jaksatomovic** [reply]
Great, Thx a lot 👍🏻
